File: Bee 1012 - Area/Area.py
#Bee 1012 - Area

val = map(float, input().split(" "))
A, B, C = val

print('TRIANGULO: %.3f' % ((A * C) / 2))
print('CIRCULO: %.3f' % (3.14159 * (C * C)))
print('TRAPEZIO: %.3f' % (((A + B) * C) / 2))
print('QUADRADO: %.3f' % (B * B))
print('RETANGULO: %.3f' % (A * B))

File: Bee 1012 - Area/Area.cpp
//Bee 1012 - Area

#include <iostream>
#include<iomanip>
#include<string>
using namespace std;

int main(){
    double a, b, c;

    cin >> a >> b >> c;

    cout << "TRIANGULO: " << fixed << setprecision(3) << (a*c)/2 << endl;
    cout << "CIRCULO: " << fixed << setprecision(3) << 3.14159*(c*c) << endl;
    cout << "TRAPEZIO: " << fixed << setprecision(3) << ((a+b)*c)/2 << endl;
    cout << "QUADRADO: " << fixed << setprecision(3) << (b*b) << endl;
    cout << "RETANGULO: " << fixed << setprecision(3) << (a*b) << endl;
    
    return 0;
}